pkg/grafana/grafana.go

```diff
@@ -102,6 +109,46 @@ func DeleteByID(ctx context.Context, grafanaAPI *client.GrafanaHTTPAPI, id int64
 	return nil
 }
 
+func CreateDatasource(ctx context.Context, grafanaAPI *client.GrafanaHTTPAPI) ([]Datasource, error) {
+	logger := log.FromContext(ctx)
+
+	createdDatasources := make([]Datasource, 0)
+	mimirDatasource := Datasource{
```

Comment:
Can you move the mimirDatasource at the top like so:

var defaultDatasources := []Datasource{
   Datasource{
    // loki values
  },
  Datasource{
    // mimir values
  }, 
}

Also we are missing the specific values like [this](https://github.com/giantswarm/mimir-app/blob/44091fc2d4ae430c4127a6dcdc1be26ec8772665/helm/mimir/templates/datasource.yaml#L26) and json data, specifically the tenant id header?

Same for the loki datasource.

Reply:
Maybe we can set readonly here?

Reply:
I've began adding more fields in the datasource.